FirebasePerformance/Sources/FPRClient.m

```diff
@@ -123,8 +123,10 @@ - (BOOL)startWithConfiguration:(FPRConfiguration *)config error:(NSError *__auto
     // Create the Logger for the Perf SDK events to be sent to Google Data Transport.
     self.gdtLogger = [[FPRGDTCCLogger alloc] initWithLogSource:logSource];
 
+#if __has_include("CoreTelephony/CTTelephonyNetworkInfo.h")
     // Create telephony network information object ahead of time to avoid runtime delays.
     FPRNetworkInfo();
```

Comment:
Good call, what's the best way to do this? If we run a tvOS sample and the events simply show up in the console, is that sufficient?

Reply:
Is this verification required for starting community support? If needed, can you let me know what this looks like. If it's not too easy, should I push this PR to a branch instead pending bandwidth from the perf team?

Reply:
As long as we are able to see the events generated on the console, we should be good.

Reply:
Few areas to verify would be if we are able to see the events on the console and if we are able to identify the device type properly on the console in the details page where the metrics can be filtered based on device type.

Reply:
Re: not having network information... how does it work for non-cellular iPads today? I suspect it wouldn't be any different. Perhaps we need to mimic what the empty case looks like to make sure the events don't get filtered out, but functionally there's no reason why a lack of network info should be broken.

Re: filtering based on device type. That is likely not to work at the moment but is being investigated. Along with our other SDKs, since this is "community supported", we should be fine with having these devices show up as "iOS" for now.

Reply:
Verified on the console for events generated from a TV (simulator) and all looks good.